src/bp-templates/bp-nouveau/includes/video/functions.php

```diff
@@ -111,7 +111,7 @@ function bp_nouveau_video_localize_scripts( $params = array() ) {
 		'group_album'                        => bp_is_group_albums_support_enabled(),
 		'messages_video'                     => bp_is_messages_video_support_enabled() && bb_user_can_create_video(),
 		'messages_video_active'              => bp_is_messages_video_support_enabled(),
-		'dropzone_video_message'             => __( 'Drop videos here to upload', 'buddyboss' ),
+		'dropzone_video_message'             => sprintf( __( '<strong>Add Videos</strong> Or drag and drop', 'buddyboss' ) ),
```

Comment:
Still, wrong see my comment here https://github.com/buddyboss/buddyboss-platform/pull/3190/files#r789405526
```suggestion
		'dropzone_video_message'             => sprintf( __( '<strong>Add Videos</strong> Or drag and drop', 'buddyboss' ) ),
```

Reply:
@chetansatasiya This is fixed now